### PR TITLE
Feature/5112 basic supplier page

### PR DESCRIPTION
**Pull request by alibombaye**
Story [AB#5112](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/5112)
Task [AB#7105](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7105)

Screenshot 2020-05-27 at 13 57 42

### CODE REVIEW

File: app/pages/sections/supplier/supplier/contextCreator.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      expect(context.backLinkText).toEqual(manifest.backLinkText);
+    });
+
+    it('should construct the backLinkHref', () => {
+      const orderId = 'order-id';
+      const context = getContext({ orderId });
+      expect(context.backLinkHref).toEqual(`${baseUrl}/organisation/${orderId}/supplier/search/select`);
+    });
+
+    it('should return the title', () => {
```
Comment (kat1906) [inline app/pages/sections/supplier/supplier/contextCreator.test.js:18]: this isn't the usual format of 'should return the contents of the manifest'?

File: app/pages/sections/supplier/supplier/manifest.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "backLinkText": "Go back",
+  "title": "Supplier information for",
```
Comment (kat1906) [inline app/pages/sections/supplier/supplier/manifest.json:3]: should there be a space at the end of this string?

File: app/pages/sections/supplier/supplier/ui.test.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    .expect(await extractInnerText(button)).eql(content.saveButtonText);
+});
+
+
+test('should redirect to /organisation/order-1/supplier/search if no supplierSelected', async (t) => {
+  await pageSetup(t, true, false);
+  await t.navigateTo(pageUrl);
+
+  await t
+    .expect(getLocation()).eql('http://localhost:1234/order/organisation/order-1/supplier/search');
```
Comment (kat1906) [inline app/pages/sections/supplier/supplier/ui.test.js:124]: should there be a check for the error?

Comment (alibombaye) [reply app/pages/sections/supplier/supplier/ui.test.js:124]: Not sure what you mean here?

Comment (kat1906) [reply app/pages/sections/supplier/supplier/ui.test.js:124]: I actually don't understand what this test is doing

File: app/pages/sections/supplier/supplier/contextCreator.test.js (hunk 1 above)
Comment (alibombaye) [reply app/pages/sections/supplier/supplier/contextCreator.test.js:18]: I've been explicitly checking each field. I think think contextCreator is going to start getting pretty complex soon to handle various different states for this page

File: app/pages/sections/supplier/supplier/ui.test.js (hunk 3 above)
Comment (alibombaye) [reply app/pages/sections/supplier/supplier/ui.test.js:124]: ah ok. it should redirect to the search route if a supplierSelected from the state is not present. Doesn't throw an error

Comment (kat1906) [reply app/pages/sections/supplier/supplier/ui.test.js:124]: can you just add that to the description please?

**Pull request merged**